website/static/js/pages/project-base-page.js

```diff
@@ -27,35 +27,33 @@ $(window).unload(function(){
     return 'Unload';
 });
 
-// TODO: See if this can be removed
 $(document).ready(function() {
-    $.getJSON(node.urls.api, function(data) {    
+    $.getJSON(node.urls.api, function(data) {
         $('body').trigger('nodeLoad', data);
     });
+});
+
+$(window).scroll(function() {
+    var scrollTop = $(this).scrollTop();
+    var offset = 49;
```

Comment:
Can you make an offset constant and use that variable in both `scroll` and `resize`?